lib/lf/assembly/coomatrix.h

```diff
@@ -0,0 +1,74 @@
+/***************************************************************************
+ * LehrFEM++ - A simple C++ finite element libray for teaching
+ * Developed from 2018 at the Seminar of Applied Mathematics of ETH Zurich,
+ * lead developers Dr. R. Casagrande and Prof. R. Hiptmair
+ ***************************************************************************/
+
+/**
+ * @file
+ * @brief Rudimentary triplet based matrix format
+ * @author Ralf Hiptmair
+ * @date August 2018
+ * @copyright MIT License
+ */
+
+#ifndef _LF_COOMATRIX_H
+#define _LF_COOMATRIX_H
+
+#include <Eigen/Sparse>
+#include "assembly_types.h"
+
+namespace lf::assemble {
+/**
+ * @brief A temporaery data structure for matrix in COO format.
+ *
+ * @tparam SCALAR basic (usually atomic) scalar type for the matrix
```

Comment:
"atomic" has a very specific meaning in programming: https://en.wikipedia.org/wiki/Linearizability . I assume you didn't refer to this concept?

Reply:
Please remove "atomic".